kubernetes/onpremise/kubeadm/examples/complete/main.tf

```diff
@@ -0,0 +1,73 @@
+
+locals {
+  master = {
+    name                = "master-1"
+    public_dns          = "PUBLIC_DNS_HERE" # it can be private if you are in the dest network
+    private_dns         = "PRIVATE_DNS_HERE"
+    tls_private_key_pem = "my master node ssh private key"
+  }
+  workers = {
+    "worker-1" = {
+      "name"                = "worker-1"
+      "labels"              = tolist(["workers", ])
+      "taints"              = tolist(["workers", ])
+      "public_dns"          = "PUBLIC_DNS_HERE" # it can be private if you are in the dest network
+      "tls_private_key_pem" = "my worker-1 node ssh private key"
+    },
+    "worker-2" = {
+      "name"                = "worker-2"
+      "labels"              = tolist(["workers", ])
+      "taints"              = tolist(["workers", ])
+      "public_dns"          = "PUBLIC_DNS_HERE" # it can be private if you are in the dest network
+      "tls_private_key_pem" = "my worker-2 node ssh private key"
+    }
+  }
+  user  = "ec2-user" #default redhat user
```

Comment:
Put these variables, which have fake values, in varaibles.tf and parameters.auto.tfvars files. Explain in the README.md that the user must update the values of the configuration in the parameters file.